proto/osmosis/twap/v1beta1/query.yml

```diff
@@ -7,7 +7,7 @@ queries:
     proto_wrapper:
       default_values:
         Req.end_time: "time.Now()"
-      query_func: "k.GetArithmeticTwap"
+      query_func_name: "GetArithmeticTwap"
     cli:
       cmd: "GetArithmeticTwap"
   GetArithmeticTwapToNow:
```

Comment:
This needs to change as well if we're to change from `query_func` -> `query_func_name`